src/dbnode/storage/series/buffer.go

```diff
@@ -54,6 +54,7 @@ const (
 var (
 	timeZero           time.Time
 	errIncompleteMerge = errors.New("bucket merge did not result in only one encoder")
+	errTooManyEncoders = errors.New("too many encoders per block")
```

Comment:
do we need to expose this with the correct error typing (should it be invalid param/internal server error/etc)

please add tests to ensure we capture